component/__manifest__.py

```diff
@@ -15,5 +17,6 @@
      'python': ['cachetools'],
  },
  'data': [],
- 'installable': False,
+ 'installable': True,
```

Comment:
@guewen It should be nice to also have the development status key into the manifest
see https://odoo-community.org/page/development-status

Reply:
Right, I was not sure at which moment it should be put, is stable the correct status?

Reply:
IMO yes it's stable!